### DISCUSSION TITLE
Page path with only two chars e.g. IT, HR, ...

### DISCUSSION THREAD

**Discussion by thomasstampfl** [requarks/wiki#5606] [Help / Questions] [answered]
Hey there!

I was wondering whether it is hard-coded or a simple configuration: I'd like to have a site like IT or HR but it seems as Wiki.js requires minimum 3 chars to work properly.

Is there a specific reason to have a minimum of 3 chars? With the given examples I think 2 would also be enough.

Thanks
Thomas

**Comment by NGPixel** [comment, marked as answer]
Because two letters path are reserved for the locale prefix (e.g. `/en` for english, `/fr` for french). Otherwise there would be no way for the system to know whether you want the page IT or the locale italian.

**Reply by thomasstampfl** [reply]
The locale is the first one, so my example should be `/en/IT` or `/en/HR`

For Italian it would be `/it/IT` or `/it/HR`

Don't matter upper or lowercase.

Would something like that be possible?

Thanks

**Reply by NGPixel** [reply]
The problem is that paths without locale are also valid.

So if your base locale is English and you try to browse to `/it`. Are you trying to access the page "it" from the base locale English (`/en/it`) or the homepage of the Italian locale (`/it`, `/it/home`)? This is ambiguous and the system has no way to know which one is correct.

**Reply by thomasstampfl** [reply]
Somehow it makes sense, somehow it doesn't.
Thank you very much for the quick and extended reply.

**Reply by hiwigiwi** [reply]
If two-letter pages break, then why is it possible for me to create them in the first place? My only option was to delete the created pages (I had two pages, one called "IT", and one called "IP"). Luckily I was not too far into populating these pages with content...

**Reply by heraldodinero** [reply]
> If two-letter pages break, then why is it possible for me to create them in the first place? My only option was to delete the created pages (I had two pages, one called "IT", and one called "IP"). Luckily I was not too far into populating these pages with content...

how did you create them? I can't

**Reply by heraldodinero** [reply]
I'm surprised more people don't find an issue with this.. I love wiki.js but the fact that URLs are forced to be much longer than I require (I  do not care in the slightest about other languages) it completely kills the product for me.. It's a tough pill to swallow.. I wish I wasn't the only person but apparently nobody seems to care
